### ISSUE TITLE
Search term: Limited character length

### ISSUE THREAD

**Issue by WebworkrNet** [mediathekview/zapp#382]
If the search term exceeds a certain length, no more search results will be found.

Example:

![Screenshot_20231012_162541_Zapp](https://github.com/mediathekview/zapp/assets/43509007/ae3a8fda-06eb-4db5-98a7-17a28fe4e368)
![Screenshot_20231012_162532_Zapp](https://github.com/mediathekview/zapp/assets/43509007/5d9243b8-1cb0-4f2d-8367-f3ebec61ea7c)

**Comment by bagbag** [comment]
MediathekViewWeb (die API hinter der Mediathek in Zapp) begrenzt in der Indizierung der Einträge die Länge der einzelnen Tokens auf 25 Zeichen.

**Comment by cemrich** [comment]
Yes, it's the same behaviour as in MediathekViewWeb. There is nothing I can do in Zapp, except restricting the input field.

**Comment by WebworkrNet** [comment]
@bagbag Thanks for the background information

@cemrich Including a reference to this issue in the FAQ would be an option.
